<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1039</width>
    <height>751</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IR Raw Data Post-Processing</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>271</width>
      <height>611</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="selectFileBut">
       <property name="text">
        <string>Select File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="playVidBut">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Play Video</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Pause Video</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextFrame">
       <property name="text">
        <string>Next Frame</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="prevFrame">
       <property name="text">
        <string>Previous Frame</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Start Frame:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Stop Frame:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="startEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="stopEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_7">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Save Video from Start to Stop Frame</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveAsVideoBut">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Save Entire File as Video</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="makeTiffBut">
       <property name="text">
        <string>Generate Tiff File</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QPushButton" name="displayC">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Display Celsius</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="displayF">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Display Fahrenheit</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="cursorTempLabel">
       <property name="text">
        <string>Cursor Temp:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="maxTempLabel">
       <property name="text">
        <string>Current Max Temp:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="maxTempLocLabel">
       <property name="text">
        <string>Max Temp Loc:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="minTempLabel">
       <property name="text">
        <string>Current Min Temp: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="minTempLocLabel">
       <property name="text">
        <string>Min Temp Loc:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="currentFrameDisp">
         <property name="text">
          <string>Current Frame:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>741</width>
      <height>601</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="dispLayout"/>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>650</y>
      <width>741</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QSlider" name="sl">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="slStartF">
         <property name="text">
          <string>Please Select File</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="slMidF">
         <property name="text">
          <string>Please Select File</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="slEndF">
         <property name="text">
          <string>Please Select File</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="slStartT">
         <property name="text">
          <string>Please Select File</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="slMidT">
         <property name="text">
          <string>Please Select File</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="slEndT">
         <property name="text">
          <string>Please Select File</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>File Selected: </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="dispSelectedFile">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>601</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Use Select File Button</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
